apollo-mockserver/src/main/java/com/ctrip/framework/apollo/mockserver/ApolloTestingServer.java

```diff
@@ -148,6 +170,30 @@ private String loadConfigFor(String namespace) {
         return GSON.toJson(apolloConfig);
     }
 
+    private Properties loadPropertiesOfNamespace(String namespace) {
+        String filename = String.format("mockdata-%s.properties", namespace);
+        Object mockdataPropertiesExits = null;
+        try {
+            mockdataPropertiesExits = RESOURCES_UTILS_LOCATOR_CLEAR.invoke(RESOURCES_UTILS_LOCATOR, filename);
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            logger.error("invoke resources util locator clear failed.", e);
+        }
+        if (!Objects.isNull(mockdataPropertiesExits)) {
+            logger.debug("load {} from {}", namespace, filename);
+            return ResourceUtils.readConfigFile(filename, new Properties());
+        }
+
+        Properties prop = new Properties();
+        try {
+            LocalFileConfigRepository localFileConfigRepository = new LocalFileConfigRepository(namespace);
+            prop = (Properties)LOCAL_FILE_CACHE_LOCATOR_CLEAR.invoke(localFileConfigRepository,
+                    new Object[]{new File(CONFIG_UTIL_LOCATOR.getDefaultLocalCacheDir()), namespace});
```

Comment:
How about simply calling the getConfig method?
```suggestion
            prop = localFileConfigRepository.getConfig();;
```